src/components/cart/Cart.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import EmptyCart from './EmptyCart'
+import FilledCart from "./FilledCart";
+import './Cart.css'
+
+function Cart(props){
+
+    const { cartData, handleQuantityUpdate } = props;
+
+    return(
+        <div className="cart">
+            {cartData.length === 0 ? 
```

Comment:
Avoid ternary in this scenario, nothing wrong in it but your else condition can be long so there can be a readability problem. So for empty cart check simply use if statement and return it. Otherwise return the Cart jsx